src/tests/aes.rs

```diff
@@ -478,7 +478,7 @@ fn test_aes_operations() {
         assert_eq!(&enc[..12], enc2.as_slice());
 
         /* test that we can get correct indicators based on inputs */
-        assert_eq!(check_validation(session, 0), true);
+        assert_eq!(check_validation(session, 1), true);
```

Comment:
the separate issue needs to be filled from this to investigate if this API is really ok to use for FIPS.

Reply:
done